src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -31,46 +61,57 @@ export class PaymentRoute extends Component<PaymentProps> {
 
         <Spacer mb={3} />
 
-        <Responsive>
-          {({ xs }) => (
-            <TwoColumnLayout
-              Content={
-                <>
-                  <Join separator={<Spacer mb={3} />}>
-                    <Placeholder height="68px" name="Credit Card" />
-                    <Placeholder height="20px" name="Billing/Shipping Check" />
-                    {!xs && (
-                      <Link to={`/order2/${order.id}/review`}>
-                        <Button size="large" width="100%">
-                          Continue
-                        </Button>
-                      </Link>
+        <Elements>
+          <Responsive>
+            {({ xs }) => (
+              <TwoColumnLayout
+                Content={
+                  <>
+                    <Join separator={<Spacer mb={3} />}>
+                      <CreditCardInput />
+                      <Checkbox
+                        selected={this.state.hideBillingAddress}
+                        onSelect={hideBillingAddress =>
+                          this.setState({ hideBillingAddress })
+                        }
+                      >
+                        Billing and shipping addresses are the same
+                      </Checkbox>
+                      <Collapse open={!this.state.hideBillingAddress}>
+                        <AddressForm
+                          country={this.state.country}
+                          onUpdateName={this.onUpdateName}
+                          onUpdateAddressLine1={this.onUpdateAddressLine1}
+                          onUpdateAddressLine2={this.onUpdateAddressLine2}
+                          onUpdateCity={this.onUpdateCity}
+                          onUpdateRegion={this.onUpdateRegion}
+                          onUpdateCountry={this.onUpdateCountry}
+                          onUpdatePostalCode={this.onUpdatePostalCode}
+                        />
+                      </Collapse>
+                      {!xs && <ContinueButton order={order} />}
+                    </Join>
+                    <Spacer mb={3} />
+                  </>
+                }
+                Sidebar={
+                  <Flex flexDirection="column">
+                    <TransactionSummary order={order} mb={xs ? 2 : 3} />
```

Comment:
`xs ? 2 : 3` can be replaced with `[2, 3]` – although we're planning to fix all instances of this pattern in a later PR so feel free to leave this as-is.

Reply:
linter rule?